apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/components/template-selector/shared.scss

```diff
@@ -0,0 +1,21 @@
+$template-selector-border-color: #e2e4e7;
+$template-selector-border-color-selected: #555d66;
+$template-selector-border-color-active: #00a0d2;
+$template-selector-border-color-hover: #c9c9ca;
+$template-selector-empty-background: #fff;
```

Comment:
While it's ok to have them here, should these variables be pulled in from Gutenberg base styles or color studio? I.e.:
```scss
$template-selector-empty-background: $white;
```